src/spotlight-samples/src/SpotlightSandboxSample/SpotlightSandboxSample.js

```diff
@@ -113,6 +113,10 @@ module.exports = kind({
 			}
 		}
 	},
+	destroy: function () {
+		Spotlight.TestMode.disable();
+		this.inherited(arguments);
+	},
 	addBarracuda: function () {
```

Comment:
As a side note, I am very curious about why we chose to use "barracuda" as the kind name in this sample.